app/components/markdown/markdown_image/index.tsx

```diff
@@ -109,11 +107,12 @@ const MarkdownImage = ({
             width: originalSize.width,
             height: originalSize.height,
         } as FileInfo;
-    }, [uri, originalSize, metadata, isReplyPost, isTablet]);
+    }, []);
```

Comment:
Not sure whether we know for sure it won't change, or we don't want it to change.